ipynb/android/workloads/Android_Recents_Fling.ipynb

```diff
@@ -296,21 +289,47 @@
     "    :type n: int\n",
     "    \"\"\"\n",
     "    # Get a list of third-party packages\n",
-    "    packages = target.execute('pm list packages -3 | cut -d: -f 2')\n",
-    "    packages = packages.split('\\r\\n')\n",
-    "    # Remove empty string\n",
-    "    packages.pop()\n",
+    "    android_version = target.getprop('ro.build.version.release')\n",
+    "    if android_version >= 'N':\n",
+    "        packages = target.execute('cmd package list packages | cut -d: -f 2')\n",
+    "        packages = packages.splitlines()\n",
+    "    else:\n",
+    "        packages = target.execute('pm list packages -3 | cut -d: -f 2')\n",
+    "        packages = packages.splitlines()\n",
+    "\n",
+    "    # As a safe fallback let's use a list of standard Android AOSP apps which are always available\n",
+    "    if len(packages) < 8:\n",
+    "        packages = [\n",
+    "            'com.android.messaging',\n",
+    "            'com.android.calendar',\n",
+    "            'com.android.settings',\n",
+    "            'com.android.calculator2',\n",
+    "            'com.android.email',\n",
+    "            'com.android.music',\n",
+    "            'com.android.deskclock',\n",
+    "            'com.android.contacts',\n",
+    "        ]\n",
     "    \n",
     "    LAUNCH_CMD = 'monkey -p {} -c android.intent.category.LAUNCHER 1 '\n",
     "    \n",
     "    if n > len(packages):\n",
     "        n = len(packages)\n",
     "        \n",
-    "    logging.info('Opening {} apps...'.format(n))\n",
-    "    for app in packages[:n]:\n",
+    "    logging.info('Trying to open {} apps...'.format(n))\n",
```

Comment:
Should we keep the same format for `logging.info()` with the `%s` everywhere? (as we do in line 325)

In which case, could you change the other lines as well?